### PR TITLE
Change h3 font-weight to 100

### PR DESCRIPTION
**Pull request by lyubomir-popov**
## Done

Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/3334

## QA

- Inspect files changed. Verify value 300 changed to 100.

Screenshot: 

image

### CODE REVIEW

**Comment by spencerbygraves** [comment]
@lyubomir-popov the difference isn't as pronounced, but this seems to shift the problem down a size.

When h3 is placed above h4 after this change, I wonder if we would we get the same comment?

It doesn't block this PR but something to think about [next cycle](https://github.com/canonical-web-and-design/vanilla-framework/issues/3334).

image

**Comment by lyubomir-popov** [comment]
@spencerbygraves I know, the only way to avoid this is to stop using the thin wait unfortunately. 

In order to ensure lower headings have thinner or same thickness contours, the ratio by which we scale them needs to be at least equal to the thickness of weight 300 divided by the thickness of weight 100. That is around 1.9, which would be huge to use as a type scale ratio. I've added more details in the attached [issue](https://github.com/canonical-web-and-design/vanilla-framework/issues/3334)

**Comment by bartaz** [comment]
Not sure if we want to address this now, but the change affects large pull quotes:

Screenshot 2020-10-07 at 10 43 43

**Comment by lyubomir-popov** [comment]
@bartaz changed to this:
image 

@spencerbygraves are you ok with this fix until we have the font weights conversation? 

I'm reducing the font-size of the bylines in quotes to be effectively `h6`'s.

File: scss/_patterns_pull-quotes.scss [hunk 1]
```diff
@@ -27,7 +27,9 @@
 
     // this needs to be nested to increase the specificity
     .p-pull-quote__citation {
-      font-style: italic;
+      @extend %vf-heading-6;
+
+      // font-style: italic;
```
Comment (bartaz) [inline scss/_patterns_pull-quotes.scss:32]: If we don't need this, it should be removed.

File: scss/_patterns_pull-quotes.scss [hunk 2]
```diff
@@ -27,7 +27,9 @@
 
     // this needs to be nested to increase the specificity
     .p-pull-quote__citation {
-      font-style: italic;
+      @extend %vf-heading-6;
```
Comment (bartaz) [inline scss/_patterns_pull-quotes.scss:30]: Do we wan't the same h6 size for all the variants?

This part of SCSS file is defining common shared styles of all quotes. Specific sizes for all variants are defined below:
line 79 defines size of default variant (as h4)
line 96 defines small variant as default text size
line 114 defines large variant as h3

I don't know why this shared definition of h6 wins in specificity and overrides all of the specific styles defined later.

If we really want the same size for all variants, we should remove individual size definitions below.

**Review by bartaz — CHANGES_REQUESTED** [review]
Some changes needed around pull-quote styles

**Review by bartaz — APPROVED** [review]
LGTM, thanks

**Comment by spencerbygraves** [comment]
>are you ok with this fix until we have the font weights conversation?

@lyubomir-popov I think that would mean the large pull quote has a smaller citation than the default one. It should probably be the same size, what do you think?

**Comment by lyubomir-popov** [comment]
@spencerbygraves no they all have the same h6, I tagged you in percy on the image that shows them all.

**Pull request merged**